openwisp_monitoring/monitoring/configuration.py

```diff
@@ -0,0 +1,519 @@
+    '#e377c2',  # raspberry yogurt pink
+    '#7f7f7f',  # middle gray
+    '#bcbd22',  # curry yellow-green
+    '#17becf',  # blue-teal
+]
+
+# under discussion
+DEFAULT_METRICS = {
+    'ping': {
+        'label': _('Ping'),
+        'name': 'Ping',
+        'key': 'ping',
+        'field_name': 'reachable',
+        'related_fields': ['loss', 'rtt_min', 'rtt_max', 'rtt_avg'],
+        'charts': {
+            'uptime': {
+                'type': 'bar',
+                'title': _('Uptime'),
+                'description': _(
+                    'A value of 100% means reachable, 0% means unreachable, values in '
+                    'between 0% and 100% indicate the average reachability in the '
+                    'period observed. Obtained with the fping linux program.'
+                ),
+                'summary_labels': [_('Average uptime')],
+                'unit': '%',
+                'order': 200,
+                'colorscale': {
+                    'max': 100,
+                    'min': 0,
+                    'label': _('Reachable'),
+                    'scale': [
+                        [0, '#c13000'],
+                        # [0.33, '#ef7d2d'],
+                        [0.5, '#deed0e'],
+                        [1, '#7db201'],
+                    ],
+                    'map': [
+                        [100, '#7db201', _('Reachable')],
+                        [33, '#deed0e', _('Partly reachable')],
+                        [None, '#c13000', _('Unreachable')],
+                    ],
+                    'fixed_value': 100,
+                },
+                'query': chart_query['uptime'],
+            },
+            'packet_loss': {
+                'type': 'bar',
+                'title': _('Packet loss'),
+                'description': _(
+                    'Indicates the percentage of lost packets observed in ICMP probes. '
+                    'Obtained with the fping linux program.'
+                ),
+                'summary_labels': [_('Average packet loss')],
+                'unit': '%',
+                'colors': [DEFAULT_COLORS[3]],
+                'order': 210,
+                'query': chart_query['packet_loss'],
+            },
+            'rtt': {
+                'type': 'scatter',
+                'title': _('Round Trip Time'),
+                'description': _(
+                    'Round trip time observed in ICMP probes, measuered in milliseconds.'
+                ),
+                'summary_labels': [
+                    _('Average RTT'),
+                    _('Average Max RTT'),
+                    _('Average Min RTT'),
+                ],
+                'unit': f' {_("ms")}',
+                'order': 220,
+                'query': chart_query['rtt'],
+            },
+        },
+        'alert_settings': {'operator': '<', 'threshold': 1, 'tolerance': 0},
+        'notification': {
+            'problem': {
+                'name': 'Ping PROBLEM',
```

Comment:
what's this key used for? Is this shown to the user?

Reply:
I am not sure how it is used internally, I didn't observe it via the admin. @pandafy might be able to confirm / correct me.

Reply:
@nepython according to the [openwisp-notifications README](https://github.com/openwisp/openwisp-notifications/tree/dev#register-notification-type), this key is not required.
Please double check and remove it if it's not needed.
@pandafy anything to add?

Reply:
@nepython this is what is shown to the user. Please rename it to `verbose_name`. It was set incorrectly earlier. :sweat_smile: 
I fixed it in [#189](https://github.com/openwisp/openwisp-monitoring/pull/189/files#diff-583ce6230e4e945550ab510f1b8be8d8R154), but please make such changes here also. It will be used while [displaying notification settings](https://user-images.githubusercontent.com/32094433/89739293-2b560600-da9d-11ea-9bdb-ce92c89dc3be.gif).